packages/sdk-js/src/models/workflow.ts

```diff
@@ -144,7 +146,16 @@ class Workflow implements WorkflowProps {
       throw new Error("Trigger is undefined in fromListResponse()");
     }
 
-    const executionCount = obj.getExecutionCount ? obj.getExecutionCount() : (obj as unknown as avs_pb.ListTasksResp.Item.AsObject).executionCount;
+    let executionCount = 0;
+    const itemObj = obj as any;
+    if (typeof itemObj.getExecutionCount === 'function') {
+      executionCount = itemObj.getExecutionCount();
+    } else if (typeof itemObj.getTotalExecution === 'function') {
+      executionCount = itemObj.getTotalExecution();
+    } else {
+      const asObject = itemObj as any;
+      executionCount = asObject.executionCount || asObject.totalExecution || 0;
+    }
 
     return new Workflow({
       id: obj.getId(),
```

Comment:
This PR should not change packages/sdk-js/src/models/workflow.ts at all. Revert this file.